Automattic/wp-calypso
# Help Center: Show error if submitting ticket fails

#### Changes proposed in this Pull Request

![image](https://user-images.githubusercontent.com/52076348/169039433-cbce455a-8b8e-42d1-9e45-064d5df64f5e.png)

- Introduced an error shown to the user if the request for submitting to forums (as a free user) or sending an email fails.
- In FORUM and EMAIL the subject is now required, without that the request would fail.
- Moved `FormInputValidation` to `@automattic/components`, as we needed to use it from the help center package

#### Testing instructions



Don't be scared of the huge number of files, 99% is only changing the import! 

* checkout the branch
* `yarn dev --sync` from `apps/editing-toolkit`
* open the help center
* check that the subject is now required in EMAIL and FORUM
* `yarn start`
* check that there are no errors for the import changes ([http://calypso.localhost:3000/devdocs/design/form-fields](http://calypso.localhost:3000/devdocs/design/form-fields) could be a nice place to check)



Related to #63552
Fixes #63552
